packages/platform/types/calendars.ts

```diff
@@ -1,9 +1,12 @@
+import { ApiProperty } from "@nestjs/swagger";
 import { Type } from "class-transformer";
+import { Transform } from "class-transformer";
 import { IsNumber, IsString, IsOptional, IsArray, ValidateNested, Validate } from "class-validator";
 
 import { IsYearMonthDays } from "./validators/isYearMonthDays";
 
 export class Calendar {
+  @Transform(({ value }: { value: string }) => parseInt(value))
```

Comment:
I would check before `parseInt` if the value is passed in the first place:
```
@Transform(({ value }: { value: string }) => value && parseInt(value))
```

so that if falsey value like "null" is passed it is not converted into "NaN" - so that if validation fails it will tell that a value needs to be passed instead of "NaN" (not a number) being passed.